src/plugins/generic/qticons/iconsloaderimpl.cpp

```diff
@@ -93,5 +173,16 @@ QString IconLoaderImpl::doMoviePath(const QString &name, uint iconSize)
 	return QString();
 }
 
+QSet<QString> IconsList::missingIcons;
+
+namespace QtIcons {
+
+void registerTypes()
+{
+	qmlRegisterType<IconsList>("org.qutim.qticons", 0, 4, "IconsList");
```

Comment:
IconLoader is a service, you may access it by [Service QML Object](https://github.com/euroelessar/qutim/blob/3c57a10fe9ce8e15bad123f0db2fc5908b247b6d/src/plugins/generic/quickchat/qml/quickchat/default/ContactsView.qml#L194)